public/apps/configuration/panels/auth-view/auth-view.tsx

```diff
@@ -94,17 +112,25 @@ export function AuthView(props: AppDependencies) {
         setDataSource={setDataSource}
         selectedDataSource={dataSource}
       />
-      <EuiPageHeader>
-        <EuiTitle size="l">
-          <h1>Authentication and authorization</h1>
-        </EuiTitle>
-        {!loading && !errorFlag && props.config.ui.backend_configurable && (
-          <ExternalLinkButton
-            href={DocLinks.BackendConfigurationDoc}
-            text="Manage via config.yml"
-          />
-        )}
-      </EuiPageHeader>
+      <PageHeader
+        navigation={props.depsStart.navigation}
+        coreStart={props.coreStart}
+        controlControls={buttonData}
+        fallBackComponent={
+          <EuiPageHeader>
+            <EuiTitle size="l">
+              <h1>Authentication and authorization</h1>
+            </EuiTitle>
+            {!loading && !errorFlag && props.config.ui.backend_configurable && (
```

Comment:
Nit: this can be factored to `{!(loading || errorFlag) && ...` to avoid assertion of negatives

Reply:
I'll leave logic changes to follow up/clean up PRs, as I don't want to risk any bugs sneaking through with such a big change.